namespaces/mson-namespace.md

```diff
@@ -237,20 +234,14 @@ Enumeration type. Exclusive list of possible elements. The elements in content's
     "element": "object",
     "content": [
         {
-            "element": "member",
-            "content": {
-                "key": {
-                    "element": "string",
-                    "content": "id"
-                },
-                "value": {
-                    "element": "number",
-                    "attributes": {
-                        "default": 0
-                    },
-                    "content": null
-                }
-            }
+            "element": "number",
+            "meta": {
+                "name": "id"
+            },
+            "attributes": {
+                "default": 0
```

Comment:
Good catch! In reality this should be the same type as element's `content`. I am afraid the MSON syntax is a bit short here. Going to update `MSON Element` definition.